core/runtime/ops.rs

```diff
@@ -143,6 +143,84 @@ pub fn queue_async_op<'s>(
   None
 }
 
+#[inline]
+pub fn map_async_op_infallible<R: 'static>(
+  ctx: &OpCtx,
+  promise_id: i32,
```

Comment:
Are these two wrappers temporary or will they still be used when `op` gets removed?

Reply:
These will replace the other forms in the file (`map_async_opXXX` and `queue_fast_async_op`/`queue_async_op`), but I'll remove OpResult at that time.